Core/GDCore/IDE/Project/ArbitraryResourceWorker.cpp

```diff
@@ -97,11 +97,11 @@ void ArbitraryResourceWorker::ExposeResources(
   }
 }
 
-void ArbitraryResourceWorker::ExposeEmbeddeds(
-    gd::String& resourceName,
-    gd::ResourcesManager* resourcesManager) {
-  if (!resourcesManager) return;
+void ArbitraryResourceWorker::ExposeEmbeddeds(gd::String& resourceName) {
+  if (resourcesManagers.empty()) return;
+  gd::ResourcesManager* resourcesManager = resourcesManagers[0];

```

Comment:
How did I missed a global list of resourcesManagers?...

Reply:
To be fair when I saw it I was thinking of removing it! This was already used by some compatibility code.

I think this should be refactored to be just a single pointer, rather than a list which does not really make sense (you can only have a single resourcesManager, one per project).
I might simplify this later, but for this PR I guess it's good enough... I wonder if we could even do WITHOUT any reference to resourcesManager.
We would "simply" have to store the kind of the embedded resource next to its name.

Reply:
It was a question I did asked myself... why there several resources managers? So now, I have my answer 😃